coralproject/talk
# Preventing backend from sending banned words to frontend

## What does this PR do?

Adds a concept of public settings so that sensitive settings can be hidden from the frontend.

## How do I test this PR?
1. List some banned words in the admin panel.
2. Load the comment stream.
3. Check the network request from the stream endpoint. The 'wordlist' array should be blank.
